.travis.yml

```diff
@@ -5,6 +5,12 @@ env:
   matrix:
   - TESTS=OGC_TESTS
   - TESTS=API
+  global:
+  - SSH_KEY="id_rsa"
+  - GIT_NAME="Isabel Rios"
+  - GIT_EMAIL="isabelrios@gmail.com"
+  - SOURCE_DIR="/home/travis/build/mozilla-sensorweb/sensorthings/test/TE_BASE/users/travis/s0001/html/sta10-0.8-SNAPSHOT.html"
```

Comment:
Yes, that will be better.

I am having troubles as travis says that:

> SOURCE_DIR (/home/travis/build/mozilla-sensorweb/sensorthings/test/TE_BASE/users/travis/s0001/html/sta10-0.8-SNAPSHOT.html) does not exist, build the source directory before deploying

So I am creating it after launching the tests to be sure the dir is already created.
Not sure if that will work or if there is a better solution